app/Http/Controllers/HomeController.php

```diff
@@ -166,10 +166,10 @@ public function quickSearch()
      *
      * #### SearchResult&lt;T>
      *
-     * Field | Type   | Description
-     * ----- | ------ | -----------
-     * data  | T[]    | |
-     * total | number | |
+     * Field | Type    | Description
+     * ----- | ------- | -----------
+     * data  | T[]     | |
+     * total | integer | |
      *
      * @queryParam mode Either `all`, `user`, or `wiki_page`. Default is `all`. Example: all
```

Comment:
```suggestion
     * @queryParam mode string Either `all`, `user`, or `wiki_page`. Default is `all`. Example: all
```
Without this it defaults to integer